src/collections/_documentation/data-management/rollups.md

```diff
@@ -74,3 +70,200 @@ A generic error, such as a database connection error, has many different stack t
 The following example will just completely overwrite Sentry's grouping by omitting `{% raw %}{{ default }}{% endraw %}` from the array:
 
 {% include components/platform_content.html content_dir='fingerprint-database-connection-example' %}
+
+## Grouping Algorithm Versioning
+
+When a Sentry project is created the latest and greatest version of the grouping algorithm is automatically selected.  However we will improve the grouping system going forward but only new projects will see these improvements not to create new groups for you.  If you do want to upgrade an existing project to a new grouping algorithm version you can do so in the project settings.  Note that you can only upgrade.  To downgrade you will need to contract customer support.
```

Comment:
```suggestion
We are constantly improving the grouping algorithm so that you don't need to see more issues than there really is.
When a Sentry project is created the latest and greatest version of the grouping algorithm is automatically selected.  However the next versions of grouping algorithm won't be applied to the projects automatically since the changes in the algorithm may generate different groups than you are used to. 
If you do want to upgrade an existing project to a new grouping algorithm version you can do so in the project settings.  Note that you can only upgrade.  
To downgrade you will need to contract customer support.
```